File: demoGit/src/main/java/model/Personne.java
package model;

public class Personne {

	private String nom;
	private String prenom;
	
	public Personne(String nom, String prenom) {
		this.prenom=prenom;
		this.nom = nom;
	}

	public String getNom() {
		return nom;
	}

	public void setNom(String nom) {
		this.nom = nom;
	}
	
	
	
}

File: olivier/angular/formationAngular/src/app/notions/model/personne.ts
export class Personne {
  constructor(private _prenom?: string, private _nom?: string) {}

  public get prenom(): string | undefined {
    return this._prenom;
  }

  public get nom(): string | undefined {
    return this._nom;
  }

  public set prenom(value: string | undefined) {
    this._prenom = value;
  }

  public set nom(value: string | undefined) {
    this._nom = value;
  }
}
